syntax = "proto3";

package fbsrankings.messages.event;

message SeasonCreatedEvent {
    string event_id = 1;
    string season_id = 2;
    int32 year = 3;
}
